src/components/DummyData.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { loadBookmarks } from '../lib/backend';
+
+export default class DummyData extends React.Component {
```

Comment:
This file, along with `lib/backend.js` can be deleted from this branch since [`fetchBookmarks`](https://github.com/beyondhb1079/s4us/blob/fetchBookmarks/src/components/DummyData.jsx) has these changes

Reply:
ok